specs/binding-mqtt-kafka.spec/src/main/scripts/io/aklivity/zilla/specs/binding/mqtt/kafka/schema/mqtt.kafka.schema.patch.json

```diff
@@ -79,15 +88,87 @@
                         ],
                         "additionalProperties": false
                     },
-                    "routes": false
+                    "routes":
+                    {
+                        "items":
+                        {
+                            "properties":
+                            {
+                                "when":
+                                {
+                                    "items":
+                                    {
+                                        "properties":
+                                        {
+                                            "subscribe":
+                                            {
+                                                "title": "Subscribe",
+                                                "type": "array",
+                                                "items":
+                                                {
+                                                    "topic":
+                                                    {
+                                                        "title": "Topic",
+                                                        "type": "string"
+                                                    }
+                                                }
+                                            },
+                                            "publish":
+                                            {
+                                                "title": "Subscribe",
+                                                "type": "array",
+                                                "items":
+                                                {
+                                                    "topic":
+                                                    {
+                                                        "title": "Topic",
+                                                        "type": "string"
+                                                    }
+                                                }
+                                            }
```

Comment:
We definitely don't need to do it now, but if we get requests for simpler syntax to avoid specifying topic once each for publish and subscribe, we can potentially add `topic` as a peer of `publish` and `subscribe` (later) that would imply both `publish` and `subscribe`.

I'd like to wait and see if it turns out we need more properties than just `topic` under `publish` and `subscribe` first, because that would impact the simplified syntax, if we need it.

Reply:
Actually, I was thinking of this. And in the regular topic space PR the syntax is already done.